extensions/wikia/CommunityPage/styles/components/_help.scss

```diff
@@ -6,23 +6,26 @@
 $small-icon-size: 18px;
 $big-icon-size: 30px;
 $help-module-margin: 12px;
-$help-module-padding-bottom: 20px;
+$help-module-padding: 24px;
+$help-module-padding-side: 18px;
 $help-module-link-padding: 12px;
+$help-module-title-padding: 3px;
 $policy-link-padding-top: 18px;
 $policy-link-margin-top: 14px;
```

Comment:
I think it would be good to make padding and margin classes (e.g. `.p-0 {padding: 0;}
.p-sm {padding: 12px;}
.p-md {padding: 18px;}` 
as global helpers and then just use them on elements